packages/ui/molecules/Hero/Hero.twig

```diff
@@ -0,0 +1,143 @@
+{#
+/**
+ * @file
+ * Hero component.
+ *
+ * @param string $surtitle
+ * @param string $title
+ * @param string $subtitle
+ * @param array $ctas[]
+ * @param string $cta_url
+ * @param string $cta_label
+ *
+ * @param array                                     $attr
+ *   customize the root element attributes.
+ * @param array<Figure> $image
+ *   Image to display. See Figure.twig for the composition.
+ * @param array                                     $image_attr
+ *   customize the image element attributes.
+ * @param array                                     $content_attr
+ *   customize the content classes.
+ * @param 'left'|'center'|'right'                   $content_position
+ *   define the content position.
+ * @param array                                     $surtitle_tag
+ *   customize the tag of the surtitle element
+ * @param array                                     $surtitle_attr
+ *   customize the surtitle element attributes.
+ * @param array                                     $title_tag
+ *   customize the tag of the title element
+ * @param array                                     $title_attr
+ *   customize the title element attributes.
+ * @param array                                     $description_tag
+ *   customize the tag of the description element
+ * @param array                                     $description_attr
+ *   customize the description element attributes.
+ * @param array                                     $cta_attr
+ *   customize the cta element attributes.
+ */
+#}
+
+{% set attributes =
+  merge_html_attributes(attr ?? null, { class: ['hero', 'w-screen', 'h-screen', 'relative'] })
+%}
+
+{# Default images attributes #}
+{% set image_attributes =
+  merge_html_attributes(
+    image_attr ?? null,
+    {
+      class: ['hero__image', 'object-cover', 'w-full', 'h-full', 'absolute', 'z-under'],
```

Comment:
**nitpick:** Remove the `object-cover` class (which is not necessary here) and replace `w-full` and `h-full` with `inset-0` to make the element take the whole available space.

```suggestion
      class: 'hero__image absolute inset-0',
```

We can drop the `z-under` class as well by adding a `relative` class to the `content_attributes`. See [this Codepen](https://codepen.io/titouanmathis/pen/gOKyLoo) for a demo of this behavior.

Reply:
Why do you drop the array here around the different classes and why would we keep it on others ? (i.e line 95)

Reply:
Alright ! Thanks for the answer and exemples.